nodejs/toolchain.bzl

```diff
@@ -22,6 +22,7 @@ NodeInfo = provider(
         "tool_files": """Files required in runfiles to make the nodejs executable available.
 
 May be empty if the target_tool_path points to a locally installed node binary.""",
+        "run_npm": "A template for a script that wraps npm",
```

Comment:
nit: mention that is a batch file if the platform is Windows, otherwise a shell script?